indico/modules/rb/models/reservations.py

```diff
@@ -492,15 +492,15 @@ def can_delete(self, user, allow_admin=True):
             return False
         return allow_admin and rb_is_admin(user) and (self.is_cancelled or self.is_rejected)
 
-    def create_occurrences(self, skip_conflicts, user=None):
+    def create_occurrences(self, skip_conflicts, user=None, allow_admin=True):
         ReservationOccurrence.create_series_for_reservation(self)
         db.session.flush()
 
         if user is None:
             user = self.created_by_user
 
         # Check for conflicts with nonbookable periods
-        if not rb_is_admin(user) and not self.room.can_manage(user, permission='override'):
+        if (not allow_admin or not rb_is_admin(user)) and not self.room.can_manage(user, permission='override'):
```

Comment:
What do you think about moving this part to simplify the condition?
(Can't think of a good var name though 😄)
```suggestion
        admin = allow_admin and rb_is_admin(user)
        if not admin and not self.room.can_manage(user, permission='override'):
```

Reply:
no strong opinion on it, so why not ;)